src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -263,7 +263,7 @@ function ComposerWithSuggestions(
     const [modal] = useOnyx(ONYXKEYS.MODAL);
     const [preferredSkinTone = CONST.EMOJI_DEFAULT_SKIN_TONE] = useOnyx(ONYXKEYS.PREFERRED_EMOJI_SKIN_TONE, {selector: EmojiUtils.getPreferredSkinToneIndex});
     const [editFocused] = useOnyx(ONYXKEYS.INPUT_FOCUSED);
-    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing, rulesdir/no-default-id-values
```

Comment:
This is intentional. If `parentReportID` is an empty string, the app will crash, so the rule should be ignored.

Reply:
Should we use `CONST.DEFAULT_NUMBER_ID` instead?

Reply:
That works, updated

Reply:
Can we also do that with other places inside `ReportActionCompose`, so we don't need to change `ComposerWithSuggestions`.